book/website/ethical-research/ethics-intro.md

```diff
@@ -0,0 +1,145 @@
+ On the one hand, RRI questions how scientists engage with research subjects. 
+ Consider, for example, how volunteers are treated during clinical trials. 
+ The point here is to work with data subjects in a respectful and inclusive way. 
+ One (in)famous case in which there was no respect for data subjects was the Tuskagee experiments, wherein African-American men were recruited to study the development of untreated syphilis, but the volunteers were falsely told they were receiving free treatment — the monitoring and lies persisted for forty years (CDC, 2020). 
+ On the other hand, we can question how scientists engage with fellow researchers. 
+ We can here discuss the importance of codes of conduct in open source collaborations, for example, [*The Turing Way*'s](https://the-turing-way.netlify.app/community-handbook/coc.html?highlight=code%20conduct); or — returning to “research misconduct” — how one might engage in authorship fraud. 
+ Once again, there are many examples of this sort of practice. 
+ From being insincere about who contributed to a paper because the academic incentives system almost encourages this (see Harvey, 2018) to inventing entirely fictional authors (see this piece by Marcus & Oransky, 2016), these are questions that are not commonly within the public’s purview. 
+
+ Thirdly, RRI and research integrity question the ethics of how scientific solutions are framed and shaped either by society at large (RRI) or by institutional practices in particular (research integrity). 
+ With our RRI hat on, we can ask how we communicate the validity of scientific outputs. 
+ Whilst how scientific findings are communicated is not a new question, it is worth highlighting that artificial intelligence (AI) and machine learning might be framed as delivering much more for research and innovation than what they actually can. 
+ Along these lines, we can ask about the role of “AI” as a marketing tool, as a way to grant research findings undeserved credibility (Lanier and Weyl, 2020). 
+ A more specific example is available in Nagendran et al.'s' (2020) concerns over exaggerated claims of AI outperforming doctors. 
+ Switching to our research integrity goggles, we can question how academic infrastructures shape scientific work. 
+
+ Consider, for example, how researchers must weigh the noble pursuit of *truth*, with the need to earn *credit* (on this weighing of options, see, for example, Zollman, 2018). 
+ We can also reflect on the impact of funding on scientific endeavours. 
+ Questions can be raised regarding the size of research grants (Bloch & Sorensen, 2014) and even the values that motivate the funding of some research projects and not others. 
+ Institutional norms might even encourage a “publication bias” whereby positively framed findings will be easier to get published (see Kaiser et al.’s, (2009) finding that studies showing differences in sex/gender are easier to publish than studies *not* supporting such differences).
+ 
```

Comment:
I feel like 'conflict of interest' has a specific professionalised meaning whereby e.g. a person's research on breakfast cereal is suspect because they are paid by the cereal manufacturer: I think what @lauracion means is more 'conflicting interests' or 'conflicting priorities', perhaps? And these questions probably don't belong in the introduction per se but should definitely be included in the book somewhere! Maybe in the philosophy chapter @Ismael-KG? https://github.com/alan-turing-institute/the-turing-way/issues/1235